src/Core/Dfe.Complete.Application/Projects/Queries/ListAllProjects/ListProjectsByMonths.cs

```diff
@@ -63,7 +63,7 @@ public async Task<PaginatedResult<List<ListProjectsByMonthResultModel>>> Handle(
                         var project = item.Project;
                         
                         var confirmedDate = project.SignificantDate.Value.ToString("MMM yyyy");
-                        var originalDate = project.SignificantDateHistories.Any() ? item.Project.SignificantDateHistories.FirstOrDefault()?.PreviousDate.Value.ToString("MMM yyyy") : null;
+                        var originalDate = project.SignificantDateHistories.Any() ? project.SignificantDateHistories.OrderBy(s => s.CreatedAt).FirstOrDefault()?.PreviousDate.Value.ToString("MMM yyyy") : null;
 
```

Comment:
I'd check for null dates in both line 66 and 65 just in case it is null, otherwise it will throw an exception

Reply:
Also could you pleaser use the constants defined in DateFormatConstants.cs instead of MMM yyyy